.ruby-version

```diff
@@ -1 +1 @@
-2.1.5
+2.3.0
```

Comment:
Should we use 2.3.1? https://www.ruby-lang.org/en/news/2016/04/26/ruby-2-3-1-released/

Travis might support it as well: http://rubies.travis-ci.org/